Dockerfile.viewer

```diff
@@ -0,0 +1,57 @@
+FROM raspbian/stretch
+MAINTAINER Anton Molodykh <amolodykh@screenly.io>
+
+RUN apt-get update && \
+    apt-get -y install \
+        build-essential \
+        curl \
+        git-core \
+        gstreamer0.10-plugins-good \
+        libdbus-glib-1-dev \
+        libffi-dev \
+        libqt5webkit5 \
+        libraspberrypi0 \
+        libssl-dev \
+        matchbox \
+        net-tools \
+        omxplayer \
+        psmisc \
+        python-dev \
+        python-gi \
+        python-netifaces \
+        sqlite3 \
+        x11-utils \
+        x11-xserver-utils \
+        xserver-xorg && \
+    apt-get clean
+
+# Install Python requirements
+ADD requirements.viewer.txt /tmp/requirements.txt
+RUN curl -s https://bootstrap.pypa.io/get-pip.py | python && \
```

Comment:
Do we really need this on Stretch? I would assume it is running a fairly recent version. it was a workaround on earlier version of Raspbian.

Reply:
yes, it is needed on Stretch. pip is not installed by default

Reply:
Instal the package `python-pip` install.